packages/shared/web/src/header/use-leverage-earn-links.tsx

```diff
@@ -30,7 +30,7 @@ export const usePendingValues = (value: number | undefined) => {
 
 export const useLeverageEarnLinks = () => {
   const theme = useTheme();
-  const selectedNetwork = useDefaultNetwork();
+  const selectedNetwork = useSelectedNetwork();
```

Comment:
Does this work on the landing page? There is no selected network there.

Reply:
Are you fine with having the useSelectedNetwork defaulting to mainnet if doesn't exist? Or do you want the same functionality as the useDefaultNetwork back?

Reply:
I am in favor of sticking with mainnet only from the landing page. If we do it based on the useWalletConnectedNetwork hook it will cause a refresh of the app if there is a connected wallet when the app loads.

Reply:
That's fine as long as the links work